tests/instruments/test_all_instruments.py

```diff
@@ -54,14 +54,97 @@
 init = ["ThorlabsPM100USB", "Keithley2700", "TC038", "Agilent34450A",
         "AWG401x_AWG", "AWG401x_AFG", "VARX", "HP8116A"]
 # Instruments which require more input arguments
-init.extend(["Instrument", "ATSBase"])
+name_required = ["ATSBase"]
```

Comment:
Clearer name `require_more_inputargs`?
Then the comments also become superfluous. :-)

Reply:
I just added the proper code to the ATSBase 😁